.github/workflows/benchmarks.yaml

```diff
@@ -0,0 +1,41 @@
+name: Benchmarks
```

Comment:
During this PR I'm also disabling semaphore CI and swapping frisbee to github actions